build/helm/keylime/templates/configmap.yaml

```diff
@@ -6,11 +6,15 @@ metadata:
   labels:
     {{- include "keylime.labels" . | nindent 4 }}
 data:
-
-{{- if .Values.global.database.mysql.enable }}
+{{- if or .Values.global.database.mysql.enable .Values.global.database.mysql.external }}
+  {{- if .Values.global.database.mysql.external }}  
+  KEYLIME_REGISTRAR_DATABASE_URL: "mysql+pymysql://{{ .Values.mysql.auth.externalUser }}:{{ .Values.mysql.auth.externalPassword }}@{{ .Values.mysql.auth.externalIP }}:3306/{{ .Values.mysql.auth.database }}?charset=utf8"
+  KEYLIME_VERIFIER_DATABASE_URL: "mysql+pymysql://{{ .Values.mysql.auth.externalUser }}:{{ .Values.mysql.auth.externalPassword }}@{{ .Values.mysql.auth.externalIP }}:3306/{{ .Values.mysql.auth.database }}?charset=utf8"
```

Comment:
I guess we could define also the external port as a value, just in case somebody is using a different port from 3306